app/models/miq_queue.rb

```diff
@@ -123,6 +123,66 @@ def self.put(options)
     msg
   end
 
+  # Trigger a background job
+  #
+  # target_worker:
+  #
+  # @options options [String] :class
+  # @options options [String] :instance
+  # @options options [String] :method
+  # @options options [String] :args
+  # @options options [String] :target_id (deprecated)
```

Comment:
I forget the reason we had target_id.  @lfu added it, I believe, but I can't remember for what purpose.  I believe it's supposed to be for when the instance_id is the object being acted upon, but the target_id is a different object.  After writing that, it makes me think it should just be an arg.